codegen/cmd/injection-gen/generators/reconciler_reconciler.go

```diff
@@ -367,18 +367,41 @@ func (r *reconcilerImpl) updateStatus(existing *{{.type|raw}}, desired *{{.type|
 			}
 		}
 
+		fullUpdate := false
+		statusUpdate := false
+
+		if !reflect.DeepEqual(existing.GetAnnotations(),desired.GetAnnotations()) {
```

Comment:
this are not imported correctly, see other examples

Reply:
Not sure what you mean by "not imported correctly".

Reply:
"formatted" maybe? Like this missing space after the comma for example, which would cause `go fmt` to fail on generated files.